MerlinAU.sh

```diff
@@ -1000,6 +1001,9 @@ _Init_Custom_Settings_Config_()
        sed -i "13 i FW_New_Update_Changelog_Approval=TBD" "$SETTINGSFILE"
        retCode=1
    fi
+   dos2unix "$SETTINGSFILE"
```

Comment:
Good idea! Approved! While I appreciate this improvement, I don't think it was the cause of the problem